e2e/src/hooks/after.hook.ts

```diff
@@ -9,6 +9,7 @@ const FAILED_RESET_ERROR = new Error('Failed to reset extension');
 
 After({ timeout: MEDIUM_TIMEOUT }, async () => {
   // [ Extension storages full reset ]
+  await BrowserContext.recorder.stop();
```

Comment:
Minor thing, but the comment right above this line doesn't refer to anything this line does. Either update the comment or move the line above it.

Reply:
`await BrowserContext.recorder.stop();` is moved above the comment